classes/Search.php

```diff
@@ -15,9 +16,9 @@ public function __construct() {
 
     public function display() {
         $data = array();
-        $argparser = new Search\ParseArgs();
-        $this->searchstring = $argparser->construct_search_string();
-        $this->searchkeyword = $argparser->searchkeyword;
+        $this->argparser = new Search\ParseArgs();
+        $this->searchstring = $this->argparser->construct_search_string();
+        $this->searchkeyword = $this->argparser->searchkeyword;
```

Comment:
Does `$argparser` need changing to a `$this->argparser`? It doesn't appear to be used anywhere but this function.